src/main/java/com/student/user/app/controllers/item/AddStudentController.java

```diff
@@ -86,8 +91,17 @@ protected void onAddStudClicked(ActionEvent event) {
         } catch(Exception ex) {
             ex.printStackTrace();;
         }
+        Stage dashboardStage = new Stage();
```

Comment:
implement the try/catch/finally statement

Reply:
okay copy